source/src/BNFC/Backend/Haskell/CFtoAbstract.hs

```diff
@@ -159,6 +178,7 @@ cf2Abstract Options{ lang, tokenText, generic, functor } name cf = vsep . concat
       [ [ text $ List.intercalate ", " stdClasses | hasTextualToks || hasData ]
       , [ text $ List.intercalate ", " funClasses | fun ]
       , [ text $ "Int, Maybe(..)" | defPosition ]
+      , [ text $ "fmap, fst, snd"]
```

Comment:
These functions should only be imported when used (currently, that would be under condition `functor`), otherwise compilation generates warnings with `-Wall`.